ttnn/cpp/ttnn/operations/conv/conv2d/device/multi_core_optimized_conv_sharded/optimized_conv_op_sharded_v2.cpp

```diff
@@ -961,25 +1018,27 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_sharded_v2_impl(
         if (enable_act_double_buffer) {
             num_act_cb_tiles = act_block_num_tiles_split;
             num_act_cb_second_reader_tiles = act_block_num_tiles_split_last;
-            num_act_cb_tiles = num_act_cb_tiles * 2; // double buffered
-            num_act_cb_second_reader_tiles = num_act_cb_second_reader_tiles * 2; // double buffered
+            num_act_cb_tiles = num_act_cb_tiles * 2;                              // double buffered
+            num_act_cb_second_reader_tiles = num_act_cb_second_reader_tiles * 2;  // double buffered
         } else {
             num_act_cb_tiles = act_block_num_tiles_split;
             num_act_cb_second_reader_tiles = act_block_num_tiles_split_last;
         }
     } else {
         if (enable_act_double_buffer) {
             num_act_cb_tiles = num_act_cb_tiles * 2;
-        } else if (conv_act_size_c / conv_act_c_blocks < 160 &&
-            per_core_out_matrix_height_ntiles < 22) {  // Q: where are these numbers from?
-            num_act_cb_tiles = num_act_cb_tiles * 2;   // double buffered
+        } else if (conv_act_size_c / conv_act_c_blocks < 160 && per_core_out_matrix_height_ntiles < 22) {  // Q: where
+                                                                                                           // are these
+                                                                                                           // numbers
+                                                                                                           // from?
```

Comment:
```suggestion
        } else if (conv_act_size_c / conv_act_c_blocks < 160 && per_core_out_matrix_height_ntiles < 22) {  // TODO
        //Q: where
                                                                                                           // are these
                                                                                                           // numbers
                                                                                                           // from?
```
Please add a TODO comment for future reference. 
I understand this is not part of the modifications in this PR, but leaving it as is will make it difficult for anyone to find later.